tasks/setup-tasks

```diff
@@ -2,19 +2,10 @@
 # set up configuration and secrets for running tasks
 set -eux
 
-# ensure we have a passwd entry for random UIDs
-# https://docs.openshift.com/container-platform/3.7/creating_images/guidelines.html
-if ! whoami && [ -w /etc/passwd ]; then
-    echo "user:x:$(id -u):0:random uid:/work:/sbin/nologin" >> /etc/passwd
-    export HOME=/work
-fi
-
 # Set up secrets
 if [ -d /run/secrets/tasks ]; then
     ln -snf /run/secrets/tasks/s3-keys ~/.config/cockpit-dev/s3-keys
     ln -snf /run/secrets/webhook/.config--github-token ~/.config/github-token
-    git config --global credential.helper store
```

Comment:
The commit message here is incorrect.  `image-refresh` still pushes branches, for example, but it's covered by this:

https://github.com/cockpit-project/bots/blob/6b7864a1e6d76bfd29b700cdf09a807cb5e3a55e/task/__init__.py#L287

Reply:
Thanks, adjusted the commit message. Also, this is still subject to real-life testing.